components/illustratedmessage/index.css

```diff
@@ -11,103 +11,143 @@ governing permissions and limitations under the License.
+		--highcontrast-illustrated-message-illustration-color: CanvasText;
+		--highcontrast-illustrated-message-illustration-accent-color: Highlight;
+	}
 }
 
-.spectrum-IllustratedMessage {
-  block-size: 100%;
+.spectrum-IllustratedMessage--sizeS {
+	--spectrum-illustrated-message-illustration-to-heading: var(--spectrum-spacing-200);
+	--spectrum-illustrated-message-illustration-to-content: var(--spectrum-spacing-200);
+	--spectrum-illustrated-message-title-font-size: var(--spectrum-illustrated-message-small-title-font-size);
+	--spectrum-illustrated-message-description-font-size: var(--spectrum-illustrated-message-small-body-font-size);
+	--spectrum-illustrated-message-cjk-title-font-size: var(--spectrum-illustrated-message-small-cjk-title-font-size);
+}
 
-  display: var(--mod-illustrated-message-display, flex);
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
+.spectrum-IllustratedMessage--sizeL {
+	--spectrum-illustrated-message-illustration-size: 160px;
+	--spectrum-illustrated-message-illustration-to-heading: var(--spectrum-spacing-100);
+	--spectrum-illustrated-message-illustration-to-content: var(--spectrum-spacing-100);
+	--spectrum-illustrated-message-title-font-size: var(--spectrum-illustrated-message-large-title-font-size);
+	--spectrum-illustrated-message-description-font-size: var(--spectrum-illustrated-message-large-body-font-size);
+	--spectrum-illustrated-message-cjk-title-font-size: var(--spectrum-illustrated-message-large-cjk-title-font-size);
+}
 
-  text-align: center;
-  pointer-events: var(--mod-illustrated-message-pointer-events, auto);
-  max-inline-size: var(--mod-illustrated-message-content-maximum-width);
+.spectrum-IllustratedMessage {
+	block-size: 100%;
+	display: var(--mod-illustrated-message-display, flex);
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+	pointer-events: var(--mod-illustrated-message-pointer-events, auto);
+	max-inline-size: var(--mod-illustrated-message-content-maximum-width);
 }
 
 .spectrum-IllustratedMessage-illustration {
-  margin-block-end: var(--mod-illustrated-message-title-to-heading, var(--spectrum-illustrated-message-title-to-heading));
-  color: var(--highcontrast-illustrated-message-illustration-color, var(--mod-illustrated-message-illustration-color, var(--spectrum-illustrated-message-illustration-color)));
-  fill: currentColor;
-  stroke: currentColor;
+	margin-block-end: var(--mod-illustrated-message-title-to-heading, var(--spectrum-illustrated-message-illustration-to-heading));
+	color: var(--highcontrast-illustrated-message-illustration-color, var(--mod-illustrated-message-illustration-color, var(--spectrum-illustrated-message-illustration-color)));
+	fill: currentColor;
+	stroke: currentColor;
+	block-size: var(--spectrum-illustrated-message-illustration-size);
+	inline-size: var(--spectrum-illustrated-message-illustration-size);
 }
 
 .spectrum-IllustratedMessage-accent {
```

Comment:
Checked in with design. Accent color is mainly used for the illustration and hover state in Drop zone. Illustrated message doesn't have accent color tokens

Reply:
Looking at the Figma specs for Drop Zone, it doesn't look like that accent on the illustration (by which I mean the little arrow that's blue where the rest of the illustration is gray) is in that either... we also don't use the accent color on drop zone now, the illustration is either all gray or all blue if it's dragged.

It might be worth double-checking to see if that means the two-colored illustration is going away. If the illustrations in S2 are meant to always be a single color (whether blue or gray), we could probably take this `.spectrum-IllustratedMessage-accent` class out, update the template and controls (and changeset notes), and update the illustration to the cloud seen in Figma (I think we can just copy the SVG), and make sure that at some point we update the Docs page too.

Illustrated message (main, with accent):
![image](https://github.com/user-attachments/assets/156b5606-b419-434c-8a49-dfd4e5d4d392)
Drop zone (main, not dragged):
![image](https://github.com/user-attachments/assets/1b4f2c7a-93a3-4a46-8097-f6e7e6ed6daf)
Drop zone (main, dragged):
![image](https://github.com/user-attachments/assets/d04d3057-34a4-4fc2-8dc5-0dc502b89d9d)

Reply:
@rise-erpelding Two colored is going away and accent color is in Dropzone when dragged.